modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/action/executor/test/ObjectActionExecutorRegistryImplTest.java

```diff
@@ -0,0 +1,154 @@
+import com.liferay.portal.kernel.util.HashMapDictionary;
+import com.liferay.portal.kernel.util.SetUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.framework.ServiceRegistration;
+
+/**
+ * @author Murilo Stodolni
+ */
+@RunWith(Arquillian.class)
+public class ObjectActionExecutorRegistryImplTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		_objectActionExecutorKey = "test-object-action";
+
+		_objectDefinition = ObjectDefinitionTestUtil.addObjectDefinition(
+			_objectDefinitionLocalService);
+
+		_systemObjectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinitionByClassName(
+				TestPropsValues.getCompanyId(), User.class.getName());
+
+		Bundle bundle = FrameworkUtil.getBundle(
+			ObjectRelationshipLocalServiceTest.class);
+
+		BundleContext bundleContext = bundle.getBundleContext();
+
+		_serviceRegistration = bundleContext.registerService(
+			ObjectActionExecutor.class,
+			new TestObjectActionExecutorImpl(
+				SetUtil.fromArray(_objectDefinition.getName()),
+				_objectActionExecutorKey),
+			new HashMapDictionary<>());
+	}
+
+	@AfterClass
+	public static void tearDownClass() throws Exception {
+		_serviceRegistration.unregister();
+
+		_objectDefinitionLocalService.deleteObjectDefinition(
+			_objectDefinition.getObjectDefinitionId());
+	}
+
+	@Test
+	public void testGetObjectActionExecutor() {
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_ADD_OBJECT_ENTRY));
+
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_GROOVY));
+
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_NOTIFICATION));
+
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_UPDATE_OBJECT_ENTRY));
+
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_WEBHOOK));
+
+		Assert.assertNotNull(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				_objectActionExecutorKey));
+	}
+
+	@Test
+	public void testGetObjectActionExecutors() {
+		List<ObjectActionExecutor> objectActionExecutors =
+			_objectActionExecutorRegistry.getObjectActionExecutors(
+				_systemObjectDefinition.getName());
```

Comment:
@murilostodolni can we declare the `_systemObjectDefinition` here instead?

```
ObjectDefinition systemObjectDefinition = _objectDefinitionLocalService.fetchObjectDefinitionByClassName(
	TestPropsValues.getCompanyId(), User.class.getName());
```

And maybe rename it to `userObjectDefinition` like you did in the other test.